src/pages-and-resources/discussions/DiscussionsSettings.test.jsx

```diff
@@ -183,22 +194,27 @@ describe('DiscussionsSettings', () => {
 
       // This is an important line that ensures the spinner has been removed - and thus our main
       // content has been loaded - prior to proceeding with our expectations.
-      await waitForElementToBeRemoved(screen.getByRole('status'));
-
-      act(async () => {
-        userEvent.click(queryByLabelText(container, 'Select Piazza'));
+      await waitForElementToBeRemoved(screen.queryByRole('status'));
 
-        userEvent.click(getByRole(container, 'button', { name: 'Next' }));
+      userEvent.click(screen.getByLabelText('Select Piazza'));
 
-        userEvent.click(await findByRole(container, 'button', { name: 'Save' }));
+      // Have to use fireEvent.click with these Stepper buttons so that the
+      // onClick handler is triggered. (userEvent.click doesn't trigger onClick).
+      await act(async () => {
+        fireEvent.click(getByRole(container, 'button', { name: 'Next' }));
+      });
+      await act(async () => {
+        fireEvent.click(getByRole(container, 'button', { name: 'Save' }));
+      });
```

Comment:
I wasn't able to remove the `act` wrappers for these `fireEvent`s -- it makes the tests fail. Any idea why?

Reply:
I won't know without digging deeper, but I think it's okay to leave as is.